mappings/net/minecraft/server/command/CloneCommand.mapping

```diff
@@ -29,3 +82,7 @@ CLASS net/minecraft/class_3023 net/minecraft/server/command/CloneCommand
 		METHOD <init> (Ljava/lang/String;IZ)V
 			ARG 3 allowsOverlap
 		METHOD method_13109 allowsOverlap ()Z
+	CLASS class_8009 ThrowingFunction
```

Comment:
Throwing is not the major purpose. It instead is a parser function. `ParserFunction`?

Reply:
I'll have to check that again. It was one of the first things I named